01_install_requirements.sh

```diff
@@ -50,9 +50,10 @@ case $DISTRO in
     # install network-scripts package to be able to use legacy network commands
     sudo dnf install -y network-scripts
     if [[ $DISTRO == "centos8" ]]; then
-      echo "CentOS is not supported anymore. Please switch to CentOS Stream / RHEL / Rocky Linux"
-    fi
-    if [[ $DISTRO == "centos8" || $DISTRO == "almalinux8" || $DISTRO == "rocky8" ]]; then
```

Comment:
I think we lost the rocky and alma part of this conditional.

Reply:
yep, lost it during the rebase.... thanks!